### PR TITLE
feat: Add DaCo Statement Prefix and Obsolete Statement support #1126

### PR DESCRIPTION
**Pull request by sachin-shitole-paccar**

### CODE REVIEW

**Review by grianbrcom — APPROVED** [review]
Looks good, but I have one noncritical comment.

**Review by temanbrcom — APPROVED** [review]
🧮

File: server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/SortTableNode.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+package org.eclipse.lsp.cobol.core.model.tree;
+
+import lombok.Getter;
+import lombok.ToString;
+import org.eclipse.lsp.cobol.core.model.Locality;
+import org.eclipse.lsp.cobol.core.model.tree.statements.ObsoleteNode;
+
+/** The class represents the Sort Table Node. **/
+@ToString(callSuper = true)
+@Getter
```
[the remark this answers is not in the document]
Comment (sachin-shitole-paccar) [reply server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/SortTableNode.java]: Done

**Review by grianbrcom — APPROVED** [review]
:beach_umbrella: 
Thanks)

**Pull request merged**